app/display/runtime/src/main/java/org/csstudio/display/builder/runtime/app/DisplayRuntimeInstance.java

```diff
@@ -33,6 +34,7 @@
 import org.phoebus.framework.persistence.Memento;
 import org.phoebus.framework.spi.AppDescriptor;
 import org.phoebus.framework.spi.AppInstance;
+import org.phoebus.ui.application.PhoebusApplication;
```

Comment:
Does DisplayRuntimeInstance need to access PhoebusApplication? It should only be aware that it's a DockItem and not care about the higher level PhoebusApplication. Maybe this is a leftover import that's not actually used?

Reply:
Ideal!